Makefile

```diff
@@ -48,7 +48,7 @@ feature-tag:
 	docker-compose exec php composer feature -- --tags $(tag)
 
 feature-ci:
-	docker-compose exec php composer feature -- --tags "~@init&&~@external" -f pretty -o std -f junit -o output/junit
+	docker-compose exec php composer feature -- --tags "~@init&&~@external&&~@pdf" -f pretty -o std -f junit -o output/junit
```

Comment:
Looks good. But I wonder, why the mailing list test was excluded?

Reply:
Probably because we don't want to get into problems with `Mailjet` by subscribing non-existing mailAddresses.
Still since we're getting a new testMail-account for PPF, It might be a good idea to also it here.

Reply:
I see, this is also done locally. Thanks for the explanation.